miner/worker.go

```diff
@@ -1015,6 +1015,7 @@ func (w *worker) prepareWork(genParams *generateParams) (*environment, error) {
 		header.ExcessBlobGas = &excessBlobGas
 		if w.chainConfig.Parlia != nil {
 			header.WithdrawalsHash = new(common.Hash)
+			*header.WithdrawalsHash = types.EmptyWithdrawalsHash
```

Comment:
It's better to 
 header.WithdrawalsHash = &types.EmptyWithdrawalsHash directly?
 replace line 1017 and 1018